content/pt-br/docs/concepts/workloads/pods/init-containers.md

```diff
@@ -0,0 +1,266 @@
+---
+reviewers:
+- erictune
+title: Init Contêineres
+content_type: concept
+weight: 40
+---
+
+<!-- overview -->
+
+Essa página provê uma visão geral de init contêineres: especializada nos contêineres que executam antes de app contêineres em um {{< glossary_tooltip text="Pod" term_id="pod" >}}. Init contêineres podem conter utilidades ou configurações que não estão presentes em uma imagem de um app.
+
+Você pode especificar init contêineres em uma especificação de Pod ao longo de um array de `containers` (que descreve app contêineres).
+
+<!-- body -->
+
+## Entendendo init contêineres
+
+Um {{< glossary_tooltip text="Pod" term_id="pod" >}} pode ter multiplos contêineres executando apps com ele, mas ele também pode ter um ou mais init contêineres, que são executados antes dos app contêineres serem iniciados.
+
+Init contêineres são exatamente como contêineres regulares, exceto:
+
+* Init contêineres executam sempre até completarem.
+* Cada init contêiner deve completar com sucesso antes de ir para o próximo início.
+
+Se o Pod do init contêiner falhar, o kubelet reinicia repetidamente aquele init contêiner
```

Comment:
This sentence appears to be switched around. The original sentence was "if a Pod's init container fails". It denotes ownership, therefore, it would be better translated as "Se o contêiner de inicialização do Pod falhar".

Reply:
All the corrections you asked I correct I await your review again.

Reply:
Hi there, @daniso123! 👋

I don't see any changes commited to this pull request. Did you push the changes to a different branch?

Reply:
I made the corrections in this issue [pt-br] Add docs/concepts/workloads/pods/init-containers.md #39193